package.json

```diff
@@ -55,17 +57,20 @@
     "react-scroll": "^1.8.1",
     "react-toastify": "^6.1.0",
     "redux-oidc": "^4.0.0-beta1",
-    "sanitize-html": "^2.3.2",
     "testcafe": "^1.9.3",
+    "ts-node": "^9.1.1",
     "typescript": "~4.0.3",
     "use-deep-compare-effect": "^1.4.0",
     "waait": "^1.0.5",
+    "xml-js": "^1.6.11",
     "yup": "^0.29.3"
   },
   "scripts": {
+    "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "react-scripts build && gzipper compress ./build && yarn generate-sitemap",
```

Comment:
Never seen this done before like this? I thought nginx handles this when the gzipping is set? Or is this done so nginx doesn't have to to improve perf?

Reply:
It was done to improve the performance and compression level. 

By default nginx uses compression level 1 (gzip_comp_level 1;) and the file size seems to be 20% bigger (in my local tests, see images) than with gzipper which uses compression level 6. I can change the compression level to 6 but it has effect to the time used for the compression.

Also nginx compresses files on-the-fly (if I understand it correctly) and it takes some CPU time to do that. In local tests the response time per file seems much faster when using pre-compressed files. 

Of course, response times can vary quite lot so it's difficult to be 100% sure about results.

**Test results**
Using gzipper
<img width="1039" alt="Screenshot 2021-05-03 at 18 34 17" src="https://user-images.githubusercontent.com/24706814/116897675-3873e000-ac3e-11eb-9d2c-44df40349fa8.png">

Not using gzipper, nging with gzip_comp_level 1;
<img width="1440" alt="Screenshot 2021-05-03 at 18 37 03" src="https://user-images.githubusercontent.com/24706814/116898025-96a0c300-ac3e-11eb-808f-bd27542e9638.png">


Not using gzipper, nging with gzip_comp_level 6;
<img width="1253" alt="Screenshot 2021-05-03 at 18 39 38" src="https://user-images.githubusercontent.com/24706814/116898360-f5663c80-ac3e-11eb-8154-2735923b657c.png">